presto-docs/src/main/sphinx/sql/grant.rst

```diff
@@ -0,0 +1,44 @@
+=====
+GRANT
+=====
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    GRANT ( privilege [, ...] | ( ALL PRIVILEGES ) )
+    ON [ TABLE ] table_name TO ( grantee | PUBLIC )
+    [ WITH GRANT OPTION ]
+
+Description
+-----------
+
+Grants the specified privileges to the specified grantee.
+
+Specifying ``ALL PRIVILEGES`` grants ``DELETE, INSERT`` and ``SELECT`` privileges.
+
+Specifying ``PUBLIC`` grants privileges to all grantees.
+
+The optional ``WITH GRANT OPTION`` clause grants the privileges to grant the specified privileges.
+
+Grant command succeeds provided the grantor possesses the specified privileges as well as the `GRANT OPTION` for those privileges.
+
+Examples
+--------
+
+Grant ``INSERT`` and ``SELECT`` privileges on the table ``orders`` to user ``alice``::
+
+    GRANT INSERT, SELECT
+    ON orders TO alice;
+
+Grant ``SELECT`` privilege on the table ``nation`` to all the users, additionally granting the privilege to grant ``SELECT`` privilege::
+
+    GRANT SELECT
+    ON orders TO PUBLIC
+    WITH GRANT OPTION;
```

Comment:
How about:

Grant ``SELECT`` privilege on the table ``nation`` to Alice, additionally allowing Alice to grant ``SELECT`` privilege to other users::

GRANT SELECT 
ON nation to alice 
WITH GRANT OPTION